CHANGELOG.md

```diff
@@ -2,6 +2,86 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+# 10.0.0-beta.18 - April 22, 2021
+## Breaking changes ⚠️
+* ResourceOptions contains tile store instance (TileStore API). Tile store usage is enabled by default, the resource option 'tileStoreEnabled' flag is introduced to disable it.
+* Add feature sdk initialisation ([#269](https://github.com/mapbox/mapbox-maps-android/pull/269))
```

Comment:
We could rephrase this entry to be more concrete, something like:

Introduce the MapInitOptions to replace MapboxMapOptions.
Introduce CredentialsManager to help manage Mapbox access tokens.
Load the Mapbox Street style by default if user doesn't load any style before the onStart lifecycle event.

etc.

Reply:
or sorry just noticed the sections below. 🤦

Reply:
still the renaming of MapboxMapOptions to MapInitOptions is still worth mentioning here.
And the default style is used now.